script/run_browser_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -eo pipefail
+
+export ROCKET_SECRET_KEY="AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
+
+mkdir -p "target/cov/plume"
+mkdir -p "target/cov/plm"
+plm='kcov --exclude-pattern=/.cargo,/usr/lib --verify target/cov/plm plm'
```

Comment:
I'm not sure to understand what you are doing here… :thinking:

Reply:
I'm creating a "shortcut" to plm run with kcov because I run it multiple time in the following lines, and it's rather long. So when I call `$plm <something>` after, it just mean "run `plm <something>` with coverage report"

Reply:
Oh OK, I get it, it is to get coverage for plm too. Great!